README.rst

```diff
@@ -95,12 +104,10 @@ Don't forget the period at the end of the command!
     # Example
 
     # CPU
-    docker build -t preprod-mxnet:1.1.0-cpu-py3 --build-arg py_version=3 \
-    --build-arg framework_installable=mxnet-1.1.0-py2.py3-none-manylinux1_x86_64.whl -f Dockerfile.cpu .
+    docker build -t preprod-mxnet:1.1.0-cpu-py3 --build-arg py_version=3.6 -f Dockerfile.cpu .
```

Comment:
For training py_version should be just '3'

Reply:
ah, my bad. didn't realize it was just serving.

Reply:
no problem, it is kind of a random change.